routes/search.js

```diff
@@ -23,16 +26,20 @@ router.get('/', common.isAuthenticated, (req, res, next) => {
     })
 })
 
+router.get('/faq', common.isAuthenticated, (req,res,next) => {
+    res.render('faq');
+})
+
 // Get all avaialable books
 // ******** need to update this based on worldwide shippers
 function allBooks(id, callback){
-    var selectAllAvailableBooks = `SELECT UserBooks.id AS id, title AS title, author AS author, imgUrl AS imgUrl, isbn13 AS isbn, Users.country
+    var selectAllAvailableBooks = `SELECT UserBooks.id AS id, title AS title, author AS author, imgUrl AS imgUrl, isbn13 AS isbn, genre AS genre, rating AS rating, pubDate AS pubDate, Users.country
```

Comment:
We shouldn't need the AS parts of this on every column you select. That is only needed if you want to change the column return name OR you are unioning two selects with different column names in which case you would only apply it to the column names that are different.